cassandra-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/CassandraCluster.java

```diff
@@ -89,6 +106,20 @@
     private final PersistedCassandraClusterHealthCheckHistory healthCheckHistory;
     @NotNull
     private final PersistedCassandraFrameworkConfiguration configuration;
+    @NotNull
+    private final PersistedCassandraClusterJobs jobsState;
+
+    public static int getPortMapping(CassandraFrameworkConfiguration configuration, String name) {
```

Comment:
I'm not crazy about this method it creates odd coupling between this and ApiController.

We can revisit in latter though.